config/config.go

```diff
@@ -93,23 +93,24 @@ func (c *CloudPlatform) UnmarshalText(text []byte) error {
 
 // Config represents mackerel-agent's configuration file.
 type Config struct {
-	Apibase       string
-	Apikey        string
-	Root          string
-	Pidfile       string
-	Conffile      string
-	Roles         []string
-	Verbose       bool
-	Silent        bool
-	Diagnostic    bool          `toml:"diagnostic"`
-	DisplayName   string        `toml:"display_name"`
-	HostStatus    HostStatus    `toml:"host_status" conf:"parent"`
-	Disks         Disks         `toml:"disks" conf:"parent"`
-	Filesystems   Filesystems   `toml:"filesystems" conf:"parent"`
-	Interfaces    Interfaces    `toml:"interfaces"  conf:"parent"`
-	HTTPProxy     string        `toml:"http_proxy"`
-	HTTPSProxy    string        `toml:"https_proxy"`
-	CloudPlatform CloudPlatform `toml:"cloud_platform"`
+	Apibase          string
+	Apikey           string
+	Root             string
+	Pidfile          string
+	Conffile         string
+	Roles            []string
+	Verbose          bool
+	Silent           bool
+	Diagnostic       bool          `toml:"diagnostic"`
+	DisableKeepAlive bool          `toml:"disable_keep_alive"`
```

Comment:
Added `DisableKeepAlive` here

Reply:
Changed it to `DisableHttpKeepAlive`
https://github.com/mackerelio/mackerel-agent/blob/12e1b6e4691483c364a77f0f02560d12fdfeca6e/config/config.go#L105